src/components/typography/H3.tsx

```diff
@@ -1,101 +1,35 @@
-import React from "react";
```

Comment:
It seems that you've migrated from the io-app the H2 component into the H3 one.
Is it right ?

Reply:
Actually not, H2 component has the following attributes:

```
export const h2FontSize = 20;
export const h2LineHeight = 24;
```